psi4/src/psi4/libqt/qt.h

```diff
@@ -85,8 +85,8 @@ void mat_print(double **A, int rows, int cols, std::string OutFileRMR);
 
 void timer_init(void);
 void timer_done(void);
-void timer_on(const char *key);
-void timer_off(const char *key);
+void timer_on(const char *key, int thread_rank = 0);
+void timer_off(const char *key, int thread_rank = 0);
```

Comment:
Can you switch these to `std::string` over `char *`? Trying to kill off `char *` in Psi4.